Dockerfile

```diff
@@ -14,14 +14,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   python-is-python3 \
   wget \
   software-properties-common \
-
+\
```

Comment:
Docker will be unhappy if this is not fixed, can remove doesn't realllly matter for this PR